src/app/components/PortraitVideoCarousel/PortraitVideoPromo/index.tsx

```diff
@@ -2,26 +2,45 @@
 import { jsx } from '@emotion/react';
 import { PortraitVideoPromoProps } from '#app/models/types/portraitVideo';
 import Image from '#app/components/Image';
-import Heading from '../../Heading';
+import Text from '#app/components/Text';
+import { Play } from '#app/components/icons';
+import VisuallyHiddenText from '#app/components/VisuallyHiddenText';
 import styles from './index.styles';
 
+// FOCUS INDICATOR
+
 export default (item: PortraitVideoPromoProps) => {
-  const { images, headlines, onClick } = item;
+  const { images, headlines, video, onClick } = item;
 
   const imageUrl = images?.[0]?.url;
   const alt = images?.[0]?.altText || '';
   const headline = headlines?.promoHeadline || '';
+  const duration = video?.version.duration;
```

Comment:
We need to convert the duration to a timestamp (it's currently displaying the ISO8601 version). In other page types we have both `duration` (which is usually the duration in seconds & `durationISO8601` [search the repo for examples] which is the duration in the format: PT1M30S, so we should check whether the BFF is passing this value from the source, or whether we need to write a utility to convert ISO format ->  duration, before we then format the duration for displaying in the promo using this: https://github.com/bbc/simorgh/blob/latest/src/app/lib/utilities/formatDuration/index.js)

```suggestion
  const duration = video?.version.duration;
```